src/views/Entries/QualityControl/index.tsx

```diff
@@ -64,8 +48,22 @@ function QualityControl(props: QualityControlProps) {
                 leftContainerClassName={styles.left}
                 leftChild={(
                     <div className={styles.frameworkSelection}>
+                        <div>
+                            Matrix 1d
+                        </div>
+                        <TableOfContents
+                            options={matrix1dToc as MatrixTocElement[]}
```

Comment:
We shouldn't cast the type of matrix1dToc, instead the return type for getMatrix1dToc should be MatrixTocElement[]